### PR TITLE
User testing tweaks

### PR DESCRIPTION
**Pull request by hammerfunk**
Mostly copy
Removed the first financial estimate question since it was a sub question
Adding error icon to sidenav and errors

### CODE REVIEW

File: app/assets/stylesheets/frontend/views/award_form.scss [hunk 1]
```diff
@@ -11,8 +11,19 @@
...
+    background-size: 16px 16px;
+
+    // Retina
+    @media only screen and (-webkit-min-device-pixel-ratio: 2), not all, not all, not all, only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx) {
+      background-image: image-url("icon-important-sm-red-2x.png");
+    }
+
+    a {
+      background: transparent;
+      color: $error-colour !important;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/views/award_form.scss]: !important should not be used

File: app/forms/qae_2014_forms/innovation/innovation_step3.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def innovation_step3
           financial_date_pointer
         end
 
-        options :financial_year_date_changed, "Did your year-end date change during your 2 or 5 year entry period?" do
+        options :financial_year_date_changed, "Did your year-end date change during your <span class='js-entry-period-subtext'>2 or 5</span> year entry period?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step3.rb]: Line is too long. [163/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step3.rb [hunk 3]
```diff
@@ -41,7 +41,7 @@ def trade_step3
           financial_date_pointer
         end
 
-        options :financial_year_date_changed, "Did your year-end date change during your (3 or 6 year) entry period?" do
+        options :financial_year_date_changed, "Did your year-end date change during your (<span class='js-entry-period-subtext'>3 or 6</span> year) entry period?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step3.rb]: Line is too long. [165/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb [hunk 4]
```diff
@@ -23,7 +24,7 @@ def development_step3
           financial_date_pointer
         end
 
-        options :financial_year_date_changed, 'Did your year-end date change during your <span class="js-entry-period-subtext">2 or 5</span> year entry period?' do
+        options :financial_year_date_changed, "Did your year-end date change during your <span class='js-entry-period-subtext'>2 or 5</span> year entry period?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [163/100]

**Comment by jaleszek** [comment]
gonna pull and check these specs

**Comment by jaleszek** [comment]
I don't see correlation specs with this pr so good to merge

**Pull request merged**